src/client/eppo-client.spec.ts

```diff
@@ -596,13 +600,223 @@ describe('EppoClient E2E test', () => {
 
     const client = new EppoClient(storage);
     let assignment = client.getAssignment('subject-10', flagKey, { appVersion: 9 });
-    expect(assignment).toEqual(null);
+    expect(assignment).toBeNull();
     assignment = client.getAssignment('subject-10', flagKey);
-    expect(assignment).toEqual(null);
+    expect(assignment).toBeNull();
     assignment = client.getAssignment('subject-10', flagKey, { appVersion: 11 });
     expect(assignment).toEqual('control');
   });
 
+  it('returns control variation and logs status_quo if subject is in holdout', () => {
+    const entry = {
+      ...mockExperimentConfig,
+      allocations: {
+        allocation1: {
+          percentExposure: 1,
+          holdout: {
+            statusQuo: 'variation-7',
+            shipped: null,
+            percentExposure: 0.2,
+            keys: [
+              {
+                holdoutKey: 'holdout-2',
```

Comment:
Calling this `holdoutKey` to be consistent with the other keys: `allocationKey`, `variationKey`